README.md

```diff
@@ -44,7 +44,11 @@ See our [Contribution guidelines](./CONTRIBUTING.md).
 
 ## Alternatives
 
-- [XcodeGen](https://github.com/yonaskolb/XcodeGen): You can commit this JSON file into Git instead of the `.pbxproj` file. Then resolving conflicts is much easier.
+All of the alternatives below allow you to generate your Xcode projects based on a spec or manifest. You commit these files to git, and can even remove the `.xcodeproj` files from git. If you decide to keep them in git, resolving conflicts is much simpler, by generating the projects again.
```

Comment:
I'd remove the last sentence, I prefer to have a general description as the readme's real estate is expensive.